_inc/client/security/monitor.jsx

```diff
@@ -49,7 +49,9 @@ export const Monitor = withModuleSettingsFormHelpers(
 							toggleModule={ this.props.toggleModuleNow }
 						>
 							<span className="jp-form-toggle-explanation">
-								{ __( "Monitor your site's downtime" ) }
+								{ __(
+									"Get alerts if your site goes offline. We’ll let you know when it's back up, too."
```

Comment:
We have two types of apostrophes here. Maybe we can stick to `’`?

```suggestion
									'Get alerts if your site goes offline. We’ll let you know when it’s back up, too.'
```